react/ProductSummaryList.js

```diff
@@ -0,0 +1,184 @@
+import React from 'react'
+import { useQuery } from 'react-apollo'
+import { ProductListContext } from 'vtex.product-list-context'
+import { ExtensionPoint, useTreePath } from 'vtex.render-runtime'
+import { useListContext, ListContextProvider } from 'vtex.list-context'
+
+import { mapCatalogProductToProductSummary } from './utils/normalize'
+import ProductListEventCaller from './components/ProductListEventCaller'
+
+import { productSearchV2 } from 'vtex.store-resources/Queries'
+
+const ORDER_BY_OPTIONS = {
+  ORDER_BY_RELEVANCE: {
```

Comment:
since these enums are inside `ORDER_BY_OPTIONS`  I think you can remove `ORDER_BY` on the attributes so it doesn't get redundant